src/pipeline_v3/tests/integration/test_e2e_integration.py

```
@@ -78,6 +84,28 @@
 
             try:
                 # Test document addition
+                print(f"About to process document: {doc_path}")
+                # Check if OpenAI API key is available and test client creation
+                api_key_in_env = bool(os.environ.get('OPENAI_API_KEY'))
+                print(f"Environment OPENAI_API_KEY present: {api_key_in_env}")
+                if hasattr(pipeline.config, 'openai') and hasattr(pipeline.config.openai, 'api_key'):
```

Ruff (0.11.9)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)